bench/Evaluation.hs

```diff
@@ -2,31 +2,22 @@
 
 module Evaluation (benchmarks) where
 
-import           Algebra.Graph
-import           Control.Monad
```

Comment:
Did you know we didn’t have warnings enabled in the benchmarks?

Reply:
😮